<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QMainWindow" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Map provider (restart needed):</string>
    </property>
   </widget>
   <widget class="QComboBox" name="mapProviderCB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>451</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan interval (in seconds):</string>
    </property>
   </widget>
   <widget class="QComboBox" name="scanInterval_CB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>451</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>Settings</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
